proptest-regressions/conversions/chrono.txt

```diff
@@ -0,0 +1,9 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
I would like to avoid checking this in for now as well. AFAIK this file is also not stable w.r.t. dependency and toolchain updates and we have no procedure for managing it.